mmedit/core/evaluation/metrics.py

```diff
@@ -165,7 +165,7 @@ def reorder_image(img, input_order='HWC'):
     return img
 
 
-def psnr(img1, img2, crop_border=0, input_order='HWC'):
+def psnr(img1, img2, crop_border=0, input_order='HWC', color_space=None):
```

Comment:
The name "color_space" needs a better design.
"Y" is not a color (standard) space

Reply:
How about `convert_to`? This is what `niqe` used.

Reply:
convert_to is not good either, think harder

Reply:
Okay, convert_to is ok

Reply:
Since we either keep the input as it is or turn it to y-channel. How about we use a boolean variable named `to_y_channel`? Not sure if there is a need to support other color spaces in the future.

Reply:
Yes this should be extensible to other channels. Sometimes we want to compute on Hue or Lightness.
Either we do not support color conversion and extraction in this function and users need to convert them before call this function, or we leave a generic interface (no need to actually implement) so that any possible conversion is possible to implement.